inc/cachify_hdd.class.php

```diff
@@ -303,19 +303,16 @@ public static function _dir_size( $dir = '.' ) {
 	private static function _file_path( $path = null ) {
 		$prefix = is_ssl() ? 'https-' : '';
 
+		$host_parts = wp_parse_url( 'http://' . strtolower( $_SERVER['HTTP_HOST'] ) );
```

Comment:
I think this can be skipped and you can use `$_SERVER['HTTP_HOST']` directly (the same way as in `Cachify_MEMCACHED::_file_path()` method.